src/cmd/honeycomb/main.go

```diff
@@ -75,10 +75,17 @@ func main() {
 		SecondaryProvider: secondaryCertProvider,
 	}
 
+	rootCACertPool := rootCAPool(config, logger)
+
 	tlsConfig := &tls.Config{
 		GetCertificate: providerAdaptor.GetCertificate,
 		Certificates:   []tls.Certificate{*defaultCertificate},
-		RootCAs:        rootCAPool(config, logger),
+		RootCAs:        rootCACertPool,
+	}
+
+	httpProxyTransport := http.DefaultTransport
+	httpProxyTransport.(*http.Transport).TLSClientConfig = &tls.Config{
```

Comment:
This is modifying the global default - best to construct a new transport.

Reply:
I changed it to use the same creation method as `DefaultTransport`, it feels messy though.